### PR TITLE
update CommandLineUtils

### PR DESCRIPTION
**Pull request by tibel**
see https://natemcmaster.github.io/CommandLineUtils/docs/intro.html#relationship-to-microsoftextensionscommandlineutils

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <!-- Tmds.DBus.Tool dependencies -->
     <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.8.0" />
     <PackageVersion Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="4.8.0" />
-    <PackageVersion Include="Microsoft.Extensions.CommandLineUtils" Version="1.1.1" />
+	<PackageVersion Include="McMaster.Extensions.CommandLineUtils" Version="4.1.1" />
```
Comment (tmds) [inline Directory.Packages.props:17]: The indenting is off.

File: src/Tmds.DBus.Tool/CodeGenCommand.cs [hunk 2]
```diff
@@ -234,7 +234,7 @@ class CodeGenArguments
             public string OutputFileName { get; set; }
             public IEnumerable<string> SkipInterfaces { get; set; }
             public Dictionary<string, string> Interfaces { get; set; }
-            public List<string> Files { get; set; }
+            public IReadOnlyList<string> Files { get; set; }
```
Comment (tmds) [inline src/Tmds.DBus.Tool/CodeGenCommand.cs:237]: The List doesn't work with the `McMaster.Extensions.CommandLineUtils`'s package?
How did you find out?

**Comment by tmds** [comment]
@tibel the tool has no tests. Did you do some manual verification?

File: src/Tmds.DBus.Tool/CodeGenCommand.cs (hunk 2 above)
Comment (tibel) [reply src/Tmds.DBus.Tool/CodeGenCommand.cs:237]: Compiler reports:
`Cannot implicitly convert type 'System.Collections.Generic.IReadOnlyList<string?>' to 'System.Collections.Generic.List<string>'. An explicit conversion exists (are you missing a cast?)`

**Comment by tibel** [comment]
> @tibel the tool has no tests. Did you do some manual verification?

I ran some commands on a Raspberry Pi to verify the behavior unchanged.

**Comment by tmds** [comment]
lgtm, thanks!

**Pull request merged**